### ISSUE TITLE
Left axis deviation

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6608]
**Preferred term label:** Left axis deviation

**Synonyms**
LAD

**Definition (free text, please give PubMed ID)**
Left axis deviation (LAD) involves the direction of depolarisation being distorted to the left (between -30° and -90°).

**Parent term (use hpo.jax.org/app)**
Abnormality of heart axis (once available)

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
